sabnzbd/decoder.py

```diff
@@ -292,7 +292,7 @@ def is_uu_junk(line: bytes) -> bool:
             except binascii.Error as msg:
                 try:
                     # Workaround for broken uuencoders by Fredrik Lundh
-                    nbytes = (((ord(line[0]) - 32) & 63) * 4 + 5) / 3
+                    nbytes = (((line[0] - 32) & 63) * 4 + 5) // 3
```

Comment:
this now looks to align with:

https://github.com/python/cpython/blob/main/Lib/encodings/uu_codec.py#L61
>             nbytes = (((s[0]-32) & 63) * 4 + 5) // 3

Reply:
I found the same in the deprecated uu library, which is why I think the test is currently testing for a failure that the workaround is supposed to be able to handle.

I noticed cpython doesn't expect an exception to be thrown by the workaround.

With the sample nzb @Safihre supplied dropping the flac files into foobar200 they all verify, I'm not familiar with the FLAC format but it appears they use CRC8 and can contain an md5 which the files do.

Reply:
Cool. In that case let's change the test.